bigtable-hbase-1.x-parent/bigtable-hbase-1.x-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestRpcRetryBehavior.java

```diff
@@ -31,7 +31,6 @@
 import com.google.bigtable.v2.ReadModifyWriteRowResponse;
 import com.google.bigtable.v2.ReadRowsRequest;
 import com.google.bigtable.v2.ReadRowsResponse;
-import com.google.cloud.bigtable.config.Logger;
```

Comment:
Where is this file getting the Logger from w/o this import statement? Maybe the Logger was suppose to be removed like in FutureUtils.java?